e2e-tests/test_config.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LoiType} from './ground-helpers';
+
+export class TestConfig {
```

Comment:
Optionally: Rather than wrapping static fields in class, you could define these as `const` at the file (module) level. Then they could be imported directly without need for being qualified by class name.

Reply:
I had originally intended this way to allow for easy extensions if we want to have different configs for different tests, but this gets confusing quickly. I like the suggestion, done.